### PR TITLE
[SPARK-32449][ML][PySpark] Add summary to MultilayerPerceptronClassificationModel

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Add training summary to MultilayerPerceptronClassificationModel...

### Why are the changes needed?
so that user can get the training process status, such as loss value of each iteration and total iteration number.

### Does this PR introduce _any_ user-facing change?
Yes
MultilayerPerceptronClassificationModel.summary
MultilayerPerceptronClassificationModel.evaluate

### How was this patch tested?
new tests

### CODE REVIEW

File: python/pyspark/ml/classification.py [hunk 1]
```diff
@@ -51,6 +51,8 @@
            'BinaryRandomForestClassificationTrainingSummary',
            'NaiveBayes', 'NaiveBayesModel',
            'MultilayerPerceptronClassifier', 'MultilayerPerceptronClassificationModel',
+           'MultilayerPerceptronClassificationSummary',
+           'MultilayerPerceptronClassificationTrainingSummary',
```
Comment (HyukjinKwon) [inline python/pyspark/ml/classification.py:55]: @huaxingao, are these going to be documented? Might be good to list at `pyspark.ml.rst` too if so. The documentation PR (https://github.com/apache/spark/pull/29188) was merged.

**Review by srowen — COMMENTED** [review]
Looks generally OK to me

File: python/pyspark/ml/classification.py (hunk 1 above)
Comment (huaxingao) [reply python/pyspark/ml/classification.py:55]: @HyukjinKwon updated `pyspark.ml.rst`. Thanks!

**Comment by srowen** [comment]
Merged to master

**Comment by huaxingao** [comment]
Thanks!

**Pull request closed without merging**